packages/shell-evaluator/src/shell-evaluator.ts

```diff
@@ -43,7 +43,7 @@ class ShellEvaluator<EvaluationResultType = ShellResult> {
       shellApi[cmd]?.acceptsRawInput &&
       !(argv[0] ?? '').startsWith('(')
     ) {
-      return shellApi[cmd](input);
+      return shellApi[cmd](input.trim().split(' ').slice(1).join(' '));
```

Comment:
This is breaking when the whitespace after the command name is not a plain space character – it’s probably best to use the same regexp as above for consistency (just without the `g` flag):

```suggestion
      return shellApi[cmd](input.trim().replace(/\s+/, ''));
```